server/core/store.go

```diff
@@ -739,7 +739,10 @@ func IsStoreContainLabel(store *metapb.Store, key, value string) bool {
 }
 
 // IsAvailableForMinResolvedTS returns if the store is available for min resolved ts.
-func IsAvailableForMinResolvedTS(s *StoreInfo) bool {
+func IsAvailableForMinResolvedTS(s *StoreInfo, state string) bool {
+	if state == "sync_recover" {
+		return !s.IsRemoved() && !s.IsTiFlash()
+	}
 	// If a store is tombstone or no leader, it is not meaningful for min resolved ts.
 	// And we will skip tiflash, because it does not report min resolved ts.
 	return !s.IsRemoved() && !s.IsTiFlash() && s.GetLeaderCount() != 0
```

Comment:
```suggestion
func IsAvailableForMinResolvedTS(s *StoreInfo, state string) bool {
	// If a store is tombstone or no leader, it is not meaningful for min resolved ts.
	// And we will skip tiflash, because it does not report min resolved ts.
	return !s.IsRemoved() && !s.IsTiFlash()
```